packages/server/modules/stats/tests/stats.spec.js

```diff
@@ -27,8 +25,7 @@ const params = { numUsers: 25, numStreams: 30, numObjects: 100, numCommits: 100
 
 describe('Server stats services @stats-services', function () {
   before(async function () {
-    this.timeout(10000)
-
+    this.timeout(15000)
```

Comment:
Had to increase the timeout since these tests easily went over 10s with most of that time coming from the setup & seeding.

I didn't actually change any tests, tho.